inst/qml/qml_components/PredictionPerformancePriors.qml

```diff
@@ -0,0 +1,145 @@
+//
+// Copyright (C) 2013-2018 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+import QtQuick			2.12
+import JASP.Controls	1.0
+import JASP.Widgets		1.0
+import JASP				1.0
+import QtQuick.Layouts	1.3
+
+Section
+{
+	title:		qsTr("Priors")
+	expanded:	false
+	columns:	1
+	
+	property string	measure:			"OE"
+
+	Group
+	{
+
+		title: 				if(measure == "OE"){qsTr("Summary estimate (O:E Ratio)")}else{qsTr("Summary estimate (C statistic)")}
```

Comment:
if you want, you can also use a ternary here:
```suggestion
		title: 				measure === "OE" ? qsTr("Summary estimate (O:E Ratio)") :  qsTr("Summary estimate (C statistic)")
```
`===` instead of `==` might be a bit more appropriate although it doesn't really matter.

Reply:
ok, and propagated